Source/scrollrt.cpp

```diff
@@ -2393,27 +1169,23 @@ void DrawAndBlit()
 	drawpanflag = 0;
 
 	lock_buf(0);
-	if (leveltype != DTYPE_TOWN) {
-		DrawView(ViewX, ViewY);
-	} else {
-		T_DrawView(ViewX, ViewY);
-	}
-	if (ctrlPan) {
+	DrawView(ViewX, ViewY);
+	if (ctrlPan || 1) {
 		ClearCtrlPan();
 	}
-	if (drawhpflag) {
+	if (drawhpflag || 1) {

```

Comment:
Add comment to remove the `|| 1`? I understand it's useful to always render these while testing.

Reply:
Well, they actually need to always render now as. I merged the clipped, non-clipped and custom-clipped blitting code so the sprints now render into/under the menu. But maybe I can implement clipping in all sprite routines and re enstate this part of the code. My original intent was to wait with that until the SDL implementation.

Reply:
Keep it simple for now.

Reply:
.. done https://github.com/diasurgical/devilutionX/pull/361/commits/095ba96e006ac2edea50783f5c21f9641228afd7